Kontent.Ai.Management/Modules/Extensions/HttpRequestHeadersExtensions.cs

```diff
@@ -56,4 +71,74 @@ private static string GetSdkPackageId()
 
         return sdkPackageId;
     }
+
+    private static string GetSource()
+    {
+        Assembly originatingAssembly = GetOriginatingAssembly();
+
+        var attribute = originatingAssembly.GetCustomAttributes<SourceTrackingHeaderAttribute>().FirstOrDefault();
+        if (attribute != null)
+        {
+            return GenerateSourceTrackingHeaderValue(originatingAssembly, attribute);
+        }
+        return null;
+    }
+
+    private static string GetProductVersion(this Assembly assembly)
+    {
+        string sdkVersion;
+
+        if (string.IsNullOrEmpty(assembly.Location))
+        {
+            // Assembly.Location can be empty when publishing to a single file
+            // https://docs.microsoft.com/en-us/dotnet/core/deploying/single-file
+            sdkVersion = assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion;
+        }
+        else
+        {
+            try
+            {
+                var fileVersionInfo = FileVersionInfo.GetVersionInfo(assembly.Location);
+                sdkVersion = fileVersionInfo.ProductVersion;
+            }
+            catch (FileNotFoundException)
+            {
+                // Invalid Location path of assembly in Android's Xamarin release mode (unchecked "Use a shared runtime" flag)
+                // https://bugzilla.xamarin.com/show_bug.cgi?id=54678
```

Comment:
I guess this is a c&p from delivery SDK. But this link leads nowhere.

Reply:
Will fix that